docs/about.md

```diff
@@ -0,0 +1 @@
+
```

Comment:
Is this file required for read the docs to work, I saw it was removed and them added again? Not an issue though, it's not a blank page on the site anymore.

Reply:
AAAA no it may have got re-added back in with a merge or something